crates/zint/cli/src/main.rs

```diff
@@ -0,0 +1,261 @@
+use anyhow::{Context, Result};
```

Comment:
I'd suggest we can split this file into `lib`, e.g. 

- `zint/cli/src/lib.rs`
- `zint/cli/src/cmd.rs`
- ...

to make it more friend for contributors to catch on the exist loigc